lib/misc.js

```diff
@@ -65,13 +65,19 @@ exports.generateId = function (fields) {
   return base32hex.encodeBuffer(hasher.digest(), { paddingChar: '-' });
 };
 
+var userArrayFileConfig = function (env, serverSecret) {
+  return {
+    iv: `${env}-environment`.substring(0, 16),
```

Comment:
Let's chat about this today. Specifically about the initialization vector, key, and encrypting the individual user ids (as opposed to the whole file.

Reply:
have made those changes the we discussed now